### DISCUSSION TITLE
Different css between editing and viewing

### DISCUSSION THREAD

**Discussion by mopg** [ueberdosis/tiptap#3537] [General Questions]
Is there a way to have different css for when the document is being edited and when it's being viewed?

Specifically, I'm using tiptap's table extension and want the table to look different between when a user is editing the table and when it is being viewed. When the user is editing the table I want vertical lines to appear between cells, but when the document is not being edited, I want those lines to go away.

Is this possible with the table extension?

**Comment by abhishekZom** [comment]
To use different CSS styles for the tiptap editor and the view mode, you can add a class or attribute to the tiptap editor or view element that you can use to target specific styles. For example, you might add a class called .editor to the tiptap editor element, and a class called .view to the view element.

Then, in your CSS, you can use these classes to apply styles to the tiptap editor and the view mode.

**Reply by mopg** [reply]
Thanks for the reply! Could you please clarify what you mean with the view element? I'm mostly following the example at the bottom of [this page](https://tiptap.dev/examples/tables) using React, which has the element `EditorContent`, I don't see a view element there?

Or are you suggesting two different `EditorContent`s there with different css classes depending on whether the editor is editable?

**Reply by abhishekZom** [reply]
There are two components tiptap-editor and tiptap-view.

You can perhaps put a class `editor` on `tiptap-editor` component and a class 'view' on `tiptap-view`.

Then put your styles in those two classes.

P.S. I was was referring to components when I said elements
